### PR TITLE
multi: add RevokedAt field to Session

### PR DESCRIPTION
**Pull request by ellemouton**
In this PR, a new `RevokedAt` field is added to the `Session` struct and it is set to the current time when `RevokeSession` is called. Note that since it is a _new_ field, old sessions will have a zero value `RevokedAt` time and so readers of a session should still use the sesssion's `Status` field to determine if a session is revoked or not.

### CODE REVIEW

File: session/tlv.go [hunk 1]
```diff
@@ -105,6 +111,10 @@ func SerializeSession(w io.Writer, session *Session) error {
 		tlvRecords, tlv.MakePrimitiveRecord(typeCreatedAt, &createdAt),
 	)
 
+	tlvRecords = append(
+		tlvRecords, tlv.MakePrimitiveRecord(typeRevokedAt, &revokedAt),
```
Comment (guggero) [inline session/tlv.go:115]: nit: could combine this with the `append` right above it.

File: session_rpcserver.go [hunk 2]
```diff
@@ -519,6 +519,11 @@ func marshalRPCSession(sess *session.Session) (*litrpc.Session, error) {
 
 	macRecipe := marshalRPCMacaroonRecipe(sess.MacaroonRecipe)
 
+	var revokeAt uint64
```
Comment (guggero) [inline session_rpcserver.go:522]: nit: s/revokeAt/revokedAt/

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

File: session/tlv.go (hunk 1 above)
Comment (ellemouton) [reply session/tlv.go:115]: indeed! will update

Comment (ellemouton) [reply session/tlv.go]: done :)

**Review by bitromortac — APPROVED** [review]
LGTM :rocket:

**Pull request merged**